addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -254,9 +254,10 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
      * when closing the wizard.
      *
      * @private
-     * @param {Object} attachments
+     * @param {Object} event
      */
-    _onAttachmentChange: function (attachments) {
+    _onAttachmentChange: function (event) {
+        const attachments = event.data;
```

Comment:
I'm confused because the signature of this function is changed but no call to this function has been changed. Was this function not called at all in current master ? 🤔

Reply:
Yes that's exactly the issue

The call seems to have been inadvertently removed in an earlier refactor, it's called by a trigger_up so I had to pass down a reference inside the OWL components